scripts/merge_tract_cat.py

```diff
@@ -121,11 +121,12 @@ def load_tract(repo, tract, patches=None, **kwargs):
 
     merged_patch_cats = []
     for patch in patches:
-        try:
-            this_patch_merged_cat = load_patch(butler, tract, patch, **kwargs)
-        except NoResults as e:
-            print(e)
+        this_patch_merged_cat = load_patch(butler, tract, patch, **kwargs)
+        if this_patch_merged_cat == 0:
```

Comment:
@wmwv do you need a `len` here? (Again, I didn't run the script, am just guessing the behavior of `load_patch`).

Reply:
Good catch.
But given that I didn't catch this before when running test examples right now, i realize that this test isn't even necessary.  It's fine to get empty `astropy.table.Table`s back and then vstack then to still an empty `Table`.  I've removed these lines.